pallets/dmp-queue/src/migration.rs

```diff
@@ -19,16 +19,16 @@
 use crate::{Config, Configuration, Overweight, Pallet, DEFAULT_POV_SIZE};
 use frame_support::{
 	pallet_prelude::*,
-	traits::StorageVersion,
+	traits::{OnRuntimeUpgrade, StorageVersion},
 	weights::{constants::WEIGHT_REF_TIME_PER_MILLIS, Weight},
 };
 
 /// The current storage version.
-pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
```

Comment:
is is a bug or was it left on purpose below the latest migration version (see migrate_to_latest below) ?

Reply:
100% a bug